pages/partials/mathjax.ejs

```diff
@@ -0,0 +1,20 @@
+<script>
+ var MathJax = {
+     tex: {
+         inlineMath: [['$', '$'], ['\\(', '\\)']]
+     },
+     svg: {
+         fontCache: 'global'
+     },
+     loader: {
+         load: ['input/tex-base', 'output/svg']
```

Comment:
Can we default to CommonHTML output for regular pages, but still allow `pl-drawing` to call `tex2svg()`? Is this as simple as adding `output/chtml`? The only reason I bring this up is that the HTML text can be copy/pasted, unlike the SVG, and I think that people copy-paste numbers from MJ equations.

Reply:
From my limited testing it seems to only allow either one output or the other at a time. Maybe for now we can default to chtml and either conditionally check the page for the specific element, or allow the output to be changed on a question by question basis (as an option in the `info.json` perhaps)

Reply:
I played around with it a bit more and actually `chtml` doesn't seem any better than `svg` for copy/pasting (both seem pretty bad), so I think it's fine to stick with `svg`.